### ISSUE TITLE
failure with c++ compilation on Mac OS X using sage app 10-5

### ISSUE THREAD

**Issue by videlec** [flatsurf/surface-dynamics#110]
There seems to be a problem with [SageMath mac application v2.5.0 (installing sage 10-5)](https://github.com/3-manifolds/Sage_macOS/releases/tag/v2.5.0) when compiling c++ code including gmpxx. More precisely, when compiling the cython generated code from `iet_linear_family.pyx` the compilation fails with
```
fatal error: 'iosfwd' file not found
```

System details
- macOS Sequoia Version 15.2
-  Xcode 16.2

## Needed informations

- try to make an example independent of `surface_dynamics`
- full error message

**Comment by videlec** [comment]
@KaiFu2210 When you have time you could complete the informations in "Needed informations" in the issue description.

**Comment by KaiFu2210** [comment]
No problem. 
- Precise macOS version installed: macOS Sequoia Version 15.2 
- Precise version of Xcode installed: 16.2

After updating the macOS system yesterday, the system prompted me to update Xcode, and I clicked to proceed. I initially thought this action installed the full Xcode application. However, when I just tried to find the Xcode version, it turned out that I only had the command line tools installed. So I installed the full version of Xcode, and the version is 16.2. 

After that, I created a file named test_iosfwd.cpp with the following content: 
#include <iosfwd>

int main() {
    return 0;
}
I compiled it using the command: g++ test_iosfwd.cpp -o test_iosfwd 
The compilation worked successfully, with no errors or warnings. I am not sure if this implies that the previous issue (fatal error: 'iosfwd' file not found) is resolved?

**Comment by saraedum** [comment]
Yes, I think this is some strange xcode hiccup. Missing headers is what usually happens when you upgrade macOS.
